includes/Discovery.php

```diff
@@ -475,13 +476,37 @@ protected function get_twitter_metadata(): array {
 		return apply_filters( 'web_stories_story_twitter_metadata', $metadata, $post );
 	}
 
+	/**
+	 * Remove the feed_links_extra action from wp_head if on post type archive.
+	 * Workaround to remove action, until 6.1 is min require and we
+	 * can use `feed_links_extra_show_post_type_archive_feed` filter.
+	 *
+	 * @since 1.29.0
+	 */
+	public function skip_feed_links_extra(): void {
+		if ( is_post_type_archive( $this->story_post_type->get_slug() ) ) {
+			remove_action( 'wp_head', 'feed_links_extra', 3 );
+		}
```

Comment:
Instead of this extra method we can just do `if ( is_post_type_archive( $this->story_post_type->get_slug() ) ) { return; }` in `print_feed_link()` below.

Reply:
I don't like that solution personally 😄